cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,48 @@
+import PageObject from '../PageObject';
+
+class SetttingsObject extends PageObject {
```

Comment:
There's a typo in the class name `SetttingsObject`. It should be `SettingsObject`. Please correct this to ensure consistency and proper functionality.